pkg/reader/home_chain.go

```diff
@@ -19,6 +19,8 @@ type OCR3ConfigWithMeta = reader_internal.OCR3ConfigWithMeta
 
 type OCR3Config = reader_internal.OCR3Config
 
+type OCR3Node = reader_internal.OCR3Node
```

Comment:
Would love a follow up ticket to remove home chain reader from internal.

Reply:
https://smartcontract-it.atlassian.net/browse/CCIP-3642